src/master-playlist-controller.js

```diff
@@ -408,6 +408,19 @@ export class MasterPlaylistController extends videojs.EventTarget {
         bubbles: true
       });
     });
+
+    this.masterPlaylistLoader_.on('playlistunchanged', () => {
+      let updatedPlaylist = this.masterPlaylistLoader_.media();
+      let playlistOutdated = this.stuckAtPlaylistEnd_(updatedPlaylist);
+
+      // Playlist has stopped updating and we're stuck at its end. Try to
```

Comment:
I'd just move this inside of the conditional (since the phrasing considers that we already checked `playlistOutdated`).